pages/apis/rest_api/team_pipelines.md

```diff
@@ -0,0 +1,174 @@
+# Team pipelines API
+
+The team pipelines API allows users to review, create, update and destroy pipelines associated with a team in your organization.
+
+## Team pipeline data model
+
+<table class="responsive-table">
+<tbody>
+  <tr><th><code>pipeline_id</code></th><td>UUID of the associated pipeline</td></tr>
+  <tr><th><code>access_level</code></th><td>The access levels that users have to the associated pipeline - <code>read_only</code>, <code>build_and_read</code>, <code>manage_build_and_read</code></td></tr>
+  <tr><th><code>pipeline_url</code></th><td>URL of the associated pipeline on the API</td></tr>
```

Comment:
I might be missing some important details, but this feels like it could be simpler:

```suggestion
  <tr><th><code>pipeline_url</code></th><td>URL of the pipeline</td></tr>
```